plugins/coverbrowser.koplugin/bookinfomanager.lua

```diff
@@ -102,6 +102,28 @@ local BOOKINFO_COLS_SET = {
         "cover_bb_data",
     }
 
+local CONFIG_SET = {
+    "filemanager_display_mode",
+    "history_display_mode",
+    "collection_display_mode",
+    "unified_display_mode",
+    "nb_cols_portrait",
```

Comment:
Are there other settings not in that list that you are explicitely ignoring?
If no, I'd rather not have such a list (that we may forget to update and keep in sync), and just save them all (a simple sql query can probably give them all if there are not iterable outside of SQL).

Reply:
That's the full list.
We need this list to iterate over it to restore the state (inlcuding nil values that are not saved to the backup).

Reply:
I'll look into sql commands to optimize.

Reply:
No CONFIG_SET (may be less effective but okay).